facia-json/src/main/scala/com/gu/facia/client/models/Config.scala

```diff
@@ -10,6 +10,39 @@ case class Backfill(
   query: String
 )
 
+sealed trait Metadata
+
+case object Canonical extends Metadata
+
+case object Special extends Metadata
+
+case object Breaking extends Metadata
+
+
+object Metadata {
+
+  val tags: Map[String, Metadata] = Map(("Canonical", Canonical), ("Special", Special), ("Breaking", Breaking))
+
+  implicit object MetadataFormat extends Format[Metadata] {
+    def reads(json: JsValue) = {
+      (json \ "type").transform[JsString](Reads.JsStringReads) match {
+        case JsSuccess(JsString(string), _) => tags.get(string) match {
+          case Some(result) => JsSuccess(result)
+          case _ => JsError("Could not convert CollectionTag")
+        }
+        case _ => JsError("Could not convert CollectionTag")
```

Comment:
Better error messages for these?
`type` is not a string and `string` is of unknown type.